packages/protocol/contracts/governance/Election.sol

```diff
@@ -824,100 +782,11 @@ contract Election is
       _electNValidatorSignerOrAccount(minElectableValidators, maxElectableValidators, accounts);
   }
 
-  /**
-   * @notice Returns a list of elected validator with seats allocated to groups via the D'Hondt
-   *   method.
-   * @return The list of elected validator signers or accounts depending on input.
-   * @dev See https://en.wikipedia.org/wiki/D%27Hondt_method#Allocation for more information.
-   */
-  function _electNValidatorSignerOrAccount(
-    uint256 minElectableValidators,
-    uint256 maxElectableValidators,
-    bool accounts // accounts or signers
-  ) internal view returns (address[] memory) {
-    // Groups must have at least `electabilityThreshold` proportion of the total votes to be
-    // considered for the election.
-    uint256 requiredVotes = electabilityThreshold
-      .multiply(FixidityLib.newFixed(getTotalVotes()))
-      .fromFixed();
-    // Only consider groups with at least `requiredVotes` but do not consider more groups than the
-    // max number of electable validators.
-    uint256 numElectionGroups = votes.total.eligible.numElementsGreaterThan(
-      requiredVotes,
-      maxElectableValidators
-    );
-
-    address[] memory electionGroups = votes.total.eligible.headN(numElectionGroups);
-    uint256[] memory numMembers = getValidators().getGroupsNumMembers(electionGroups);
-    // Holds the number of members elected for each of the eligible validator groups.
-    uint256[] memory numMembersElected = new uint256[](electionGroups.length);
-    uint256 totalNumMembersElected = 0;
-
```

Comment:
This just got moved instead of deleted, right?

Reply:
correct. (in commit [9e31318](https://github.com/celo-org/celo-monorepo/pull/11353/commits/9e3131800f37cc8b162bdebe51a5a51e31ccc521))